docs/sphinx/users/imagej/options.rst

```diff
@@ -0,0 +1,28 @@
+Bio-Formats plugin configuration options
+========================================
+
+The Bio-Formats plugin can be configured by opening the **Bio-Formats Plugins 
+Configuration** dialog from the Plugin menu. The General tab allows for 
+configuration of features for the plugin such as upgrade checking or modifying 
+the slice label pattern.
+
+Configuring the slice label pattern
+-----------------------------------
+
+The slice label is the text displayed at the top of the image window in ImageJ.
+The values displayed here can be modified by configuring the slice label pattern.
+
+The list of available parameters for configuration is as follows:
+::
+
+    %s - series index
```

Comment:
This is fine, but you could use a definition list here, i.e.

```
%n
  series name
%c
  channel index
etc.
```

Reply:
Ive updated and opted for a field list which seemed to look better from a formatting perspective but I can swap to a definition or option list if that is more the correct implementation.